contracts/proxies/Proxy.sol

```diff
@@ -1,5 +1,7 @@
 pragma solidity ^0.5.0;
 
+/// @title IProxy - Helper interface to access masterCopy of the Proxy on-chain
+/// @author Richard Meissner - <richard@gnosis.io>
 interface IProxy {
     function masterCopy() external view returns (address);
```

Comment:
Why not making `masterCopy` variable public? Is this more optimal?

Reply:
yes since the solidity compiler generates more code

Reply:
Nice hack